### DISCUSSION TITLE
Can it bypass office365

### DISCUSSION THREAD

**Discussion by holuwadrizzy440** [postalserver/postal#2347] [Help with using Postal]
if it can bypass office365 i'll pay

**Comment by willpower232** [comment]
what do you mean by bypass?

**Comment by holuwadrizzy440** [comment]
If it can deliver to office inbox?

Accounting Dept
574-537-8900
408 S. Byrkit St
Mishawaka, IN
46544
***@***.***
https://store.lci1.com/

**Reply by willpower232** [reply]
Postal can deliver to any SMTP address so yes you can email whoever you'd like from Postal. It is of course up to yourself to set up the DNS records to verify the message and find a server provider who will allow you to send email messages.
